src/components/ShareHeader.tsx

```diff
@@ -7,9 +7,8 @@ import { openShare } from '../helpers';
 type Props = {
   headerRight?: boolean;
   // the type of shareContent should be react-native's ShareContent
+  // TODO: I guess we need a alternative shareContent for webscreen without message and title
   shareContent?: {
-    message: string;
-    title: string;
     url: string;
   };
```

Comment:
I had some difficulties testing this theory ( very disappointing wifi ) , what I believe is that `message` and `title` are required , therefore shareHeader is not working with webScreen where nieder title or message are present .